### PR TITLE
BIPT-31: As a user and a proposer, I will be able to click the pencil button to edit my proposal

### PR DESCRIPTION
**Pull request by KaceyWood8**
### Related Ticket

<https://eyblockchain.atlassian.net/jira/software/projects/BIPT/boards/301?selectedIssue=BIPT-31>

#### Dependent PR's

NA

### Description

Edit Icons were added to each draft proposal in the drafts page. The edit icon routes the user to an updateDraft form that populates all fields for that given draft. A user may update any field as desired. The save button saves all updates for that given draft.  

image

### QA Instructions
- [ ] clone repo
- [ ] npm i
- [ ] git checkout kinj/lana/kace/bipt-31
- [ ] npm run dev
- [ ] navigate to localhost:3000 in browser

### Checklist

- [x] I have reviewed the code changes myself
- [x] My code meets all of the acceptance criteria of the ticket it closes.
- [x] I have removed unused code (e.g., `console.logs`, commented out blocks, etc.)
- [ ] I have made all necessary changes to the documentation. - no changes needed
- [ ] I have added tests to cover my changes. - no tests for this ticket yet
- [ ] All new and existing tests passed. - no tests for this ticket yet
- [x] Any dependent changes have been merged and published.
- [x] Check PR title name is correct.

### CODE REVIEW

**Comment by meaghanfitzgerald** [comment]
@aj-may pls hit the magic button 😁

File: components/UpdateDraft.tsx [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+import Link from "next/link";
+import { useForm, Controller } from "react-hook-form";
+import { useRouter } from "next/router";
+import { useLeadershipSponsor } from "lib/useLeadershipSponsor";
+import { Proposal } from "@prisma/client";
+import { useUpdateDraft } from "lib/useDrafts";
+import { useEffect, useState } from "react";
+import DatePicker from "react-datepicker";
+
+import moment from "moment";
```
Comment (abdulmalik97) [inline components/UpdateDraft.tsx:24]: We aren't using moment throughout the application. Might as well remove it completely? It makes the app heavy as moment has a large codebase.

**Review by meaghanfitzgerald — CHANGES_REQUESTED** [review]
once comments are addressed, everything should be good to merge.

**Review by abdulmalik97 — APPROVED** [review]

**Review by meaghanfitzgerald — APPROVED** [review]
lgtm

**Pull request merged**